evaluations/yieldscan_1_mmagician.md

```diff
@@ -0,0 +1,44 @@
+|    0d. | Article/Tutorial           | <ul><li>[x] </li></ul> | <ul><li>[YieldScan Help Center](https://intercom.help/yieldscan)</li></ul>                                                                                                                                                                                                                                                                                                           | Tutorial videos are helpful                                                                                                                                   |
+|     1. | Controller Account Support | <ul><li>[ ] </li></ul> | https://yieldscan.app/                                                                                                                                                                                                                                                                                                                                                               |                                                                                                                                                               |
+|     2. | Ledger Wallet Support      | <ul><li>[ ] </li></ul> | https://yieldscan.app/                                                                                                                                                                                                                                                                                                                                                               |                                                                                                                                                               |
+|     3. | Docker                     | <ul><li>[x] </li></ul> | <ul><li>[Docker file](https://hub.docker.com/r/sahilnanda/yieldscan-frontend)</li><li>[Docker command](https://github.com/yieldscan/yieldscan-frontend/tree/develop#docker)</li></ul>                                                                                                                                                                                                |                                                                                                                                                               |
+
+## General notes
+
+A lot of code duplication between the crawler and the api repos, probably everything under `src/models` & `src/interfaces` & large parts of other sub-directories. Would be good to consider using for example git submodules to share the code.
```

Comment:
> A lot of code duplication between the crawler and the api repos

This is because in the early stage of YieldScan we decided to run both crawler and api as a single application/service, which after a while started creating issues for us because of faster development/changes in the core products of the polkadot ecosystem(mainly polkadot-api). Then later on we decide to separate both which created a lot of duplicate code in both the directories.

> Would be good to consider using for example git submodules to share the code.

We did remove a lot of unnecessary files and dir w.r.t the service when we separated these. Also, as per your advice we have moved remaining common part to the submodules